packages/guides/src/Interlink/InventoryLink.php

```diff
@@ -38,4 +39,14 @@ public function getTitle(): string
     {
         return $this->title;
     }
+
+    public function getOriginalKey(): string
```

Comment:
Why do we need this? It seems to be unused right now, if it's needed for a follow up please remove the methods for now, so we do not have dead code.

Reply:
This was a leftover from splitting up the commits, removed it